src/components/infinite-scroll/__tests__/InfiniteScroll.test.tsx

```diff
@@ -145,31 +150,14 @@ describe('components/infinite-scroll/InfiniteScroll', () => {
         );
 
         const sentinel = getSentinel();
-        sandbox.stub(sentinel, 'getBoundingClientRect').returns({ top: 500 + (threshold + 1) });
+        sandbox.stub(sentinel, 'getBoundingClientRect').returns({ top: 500 + (threshold + 1) } as DOMRect);
 
         scrollContainer.dispatchEvent(new Event('scroll'));
         expect(mockOnLoadMore).not.toHaveBeenCalled();
     });
 
-    it('should reset event listeners on useWindow or scrollContainerNode update', () => {
-        const component = mount(<InfiniteScroll {...propsList} />);
-        const mockAddEventListeners = jest.spyOn(component.instance(), 'addEventListeners');
-        const mockRemoveEventListeners = jest.spyOn(component.instance(), 'removeEventListeners');
-        component.update();
-
-        component.setProps({ useWindow: false });
-        expect(mockAddEventListeners).toHaveBeenCalledTimes(1);
-        expect(mockRemoveEventListeners).toHaveBeenCalledTimes(1);
-
-        component.setProps({
-            scrollContainerNode: document.createElement('div'),
-        });
-        expect(mockAddEventListeners).toHaveBeenCalledTimes(2);
-        expect(mockRemoveEventListeners).toHaveBeenCalledTimes(2);
-    });
-
     describe('with sentinel in range', () => {
-        let component;
+        let component: ReturnType<typeof mount>;
```

Comment:
It might be better to import `ReactWrapper` from enzyme although there doesn't seem to be a direct equivalent to `ShallowWrapper` for mount: https://enzymejs.github.io/enzyme/docs/api/ReactWrapper/mount.html

```suggestion
        let component: ReactWrapper;
```

Reply:
I don't care really, just needed something to make TS happy